keras_cv/layers/preprocessing/with_keypoints_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.layers import preprocessing
+
+# Imports from with_labels the list of augmentation that should perform a No-Op.
+from keras_cv.layers.preprocessing.with_labels_test import TEST_CONFIGURATIONS
+
+# List here the layers that are expected to modify keypoints with
+# their specific parameters.
+GEOMETRIC_TEST_CONFIGURATIONS = []
+
+
+class WithKeypointsTest(tf.test.TestCase, parameterized.TestCase):
+    @parameterized.named_parameters(
+        *TEST_CONFIGURATIONS,
+        *GEOMETRIC_TEST_CONFIGURATIONS,
+        ("CutMix", preprocessing.CutMix, {}),
+    )
+    def test_can_run_with_keypoints(self, layer_cls, init_args):
+        if "clip_points_to_image_size" in init_args:
```

Comment:
avoid logic in test cases.  Prefer duplicating the list of cases

Reply:
This is an old logic that I think should be removed. As I commented in #519 , I think clipping should be done with the creation of a keypoint mask, to avoid to introduce ragged tensor, which are very bad for vectorization and XLA optimization. Therefore the clipping should be done explicetly by the end-user as the layer will report all keypoints and a mask of valid keypoints.

I removed these test case from the PR head.